client/app/components/ServicesTemplate/data.json

```diff
@@ -6,48 +6,51 @@
           "id": "1",
           "title": "Name and Address",
           "content": "Verify telephone numbers, accounts, email addresses, names, SSN, and addresses",
-          "url": "telified",
+          "url": "3D",
           "img": "3D"
         },
         {
           "id": "2",
           "title": "Active / Inactive",
           "content": "Retrive additional data based on telephone, email, SSN, or address",
-          "url": "telified",
-          "img": "enrichment"
+          "url": "AI",
+          "img": "AI"
         },
         {
           "id": "3",
           "title": "Consent Re-verification",
           "content": "Use our best of breed fuzzy matching to compare multiple contact records",
-          "url": "telified",
-          "img": "comparison"
+          "url": "CR",
+          "img": "CR"
         },
         {
           "id": "4",
           "title": "Inter-Continental",
           "content": "CNAM, LIDB, Verification, and Status hosting services of the telecom industry",
-          "url": "telified",
-          "img": "telecom"
+          "url": "IC",
+          "img": "IC"
         }],
       "serviceCardContent":
         [{
           "id": "1",
           "title": "Telecom Hosting",
           "subtitle": "CNAM, LIDB & Verification. ",
-          "url":"route"
+          "url":"route",
+          "img": "telecom hosting"
         },
         {
           "id": "2",
           "title": "Wireless Flagging",
           "subtitle": "Avoid costly TCPA violations by using Data Verification Services on your data. ",
-          "url":"route"
+          "url":"route",
+          "img": "wireless flagging"
         },
         {
           "id": "3",
           "title": "Contact Data Enrichment",
           "subtitle": "Ensure you are reaching the people you mean to with accurate contacts. ",
-          "url":"route"
+          "url":"route",
+          "img": "contact data enrichment"
```

Comment:
avoiding using spaces when naming files, use snake_case convention

Reply:
I'm also confused why the routes are just `route` when I already went to the trouble of making the `telecom-hosting`, `wireless-flagging`, etc routes during our first pass of the site